src/libs/conduit/python/conduit_python.cpp

```diff
@@ -183,6 +198,35 @@ PyUnicode_From_UTF32_Unicode_Buffer(const char *unicode_buffer,
 //-----------------------------------------------------------------------------
 //-----------------------------------------------------------------------------
 
+//---------------------------------------------------------------------------//
+//---------------------------------------------------------------------------//
+// Module Code
+//---------------------------------------------------------------------------//
+//---------------------------------------------------------------------------//
+
+struct module_state {
+    PyObject *error;
+    #ifdef Py_LIMITED_API
+    PyTypeObject* PyConduit_DataType_TYPE;
+    PyTypeObject* PyConduit_Generator_TYPE;
+    PyTypeObject* PyConduit_Schema_TYPE;
+    #endif
+};
+
+//---------------------------------------------------------------------------//
+#if defined(IS_PY3K)
```

Comment:
Is Python2 really still supported?

Reply:
This is original code that was brought from bellow around line 8128